### ISSUE TITLE
Collections download

### ISSUE THREAD

**Issue by epageperron** [cdli-gh/Framework#44]
## Summary
Rewrite the collections download module to integrate in the new framework and not using a ftp client on archival

## Other links or relevant information

Collections managers (museum staff) can at this time login into the CDLI and download all images for all of their artifacts or part of it, based on the search query they do. The site creates an archive file with the required images on the archival server, shares it on a CuteFTP server through a unique link that is sent by e-mail to the collections manager.

- This module needs to be fully rewritten in cakephp
- Use Python for backend requirements
- The file to download should be accessed by the user from the production server, not archival
- Add ptm and eps files to collections downloads

#### 📈 Progress (0-1): 

See Gantt: http://cdli-dev.org/gantt/Framework/

**Comment by nomoon** [comment]
Should I track things that come up with respect to the new deployment scheme and existing archival collections downloader in a separate issue?

**Comment by epageperron** [comment]
This issue is only about writing up the new collections download module, and is dependent on #58  ;  Anything related to problems with the migration of prod (156) to the VM, like possible challenges with the current collections download should be noted elsewhere.
